### PR TITLE
feat(minify): extract `LICENSE && Copyright` comments by default (`options.extractComments`)

### PR DESCRIPTION
**Pull request by evan-scott-zocdoc**
This is the regex we've been using in our builds and it's been successfully grabbing more of the license headers.

### CODE REVIEW

File: src/uglify/minify.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ const buildComments = (options, uglifyOptions, extractedComments) => {
   // /^\**!|@preserve|@license|@cc_on/
   if (typeof options.extractComments === 'boolean') {
     condition.preserve = commentsOpts;
-    condition.extract = /^\**!|@preserve|@license|@cc_on/;
+    condition.extract = /Copyright|LICENSE|@preserve|@license|@cc_on/ig;
```
Comment (evilebottnawi) [inline src/uglify/minify.js]: Don't remove `\**!`

**Comment by evilebottnawi** [comment]
@evan-scott-zocdoc Thanks for PR

File: src/uglify/minify.js (hunk 1 above)
Comment (evan-scott-zocdoc) [reply src/uglify/minify.js]: The regex won't work without it, since not everyone uses the `/**` type of comments.

Comment (evilebottnawi) [reply src/uglify/minify.js]: @evan-scott-zocdoc we use `|` (`or`) for comments types

Comment (evan-scott-zocdoc) [reply src/uglify/minify.js]: OHH, sorry I see. Will fix it.

File: src/uglify/minify.js [hunk 2]
```diff
@@ -40,7 +40,7 @@ const buildComments = (options, uglifyOptions, extractedComments) => {
   // /^\**!|@preserve|@license|@cc_on/
   if (typeof options.extractComments === 'boolean') {
     condition.preserve = commentsOpts;
-    condition.extract = /^\**!|@preserve|@license|@cc_on/;
+    condition.extract = /^\**!|Copyright|LICENSE|@preserve|@license|@cc_on/ig;
```
Comment (michael-ciniawsky) [inline src/uglify/minify.js]: Are these `uglify-es` defaults aswell ?

Comment (evan-scott-zocdoc) [reply src/uglify/minify.js]: Not sure, this is the config we've been using at Zocdoc that extracts more of the relevant comments.

Comment (evilebottnawi) [reply src/uglify/minify.js]: @evan-scott-zocdoc maybe also do PR in uglify-es?

Comment (evan-scott-zocdoc) [reply src/uglify/minify.js]: I could, but you're already defining your own override here so me changing it in uglify-es wouldn't fix anything?

Comment (evilebottnawi) [reply src/uglify/minify.js]: @evan-scott-zocdoc First in uglify, second we merge here

Comment (michael-ciniawsky) [reply src/uglify/minify.js]: If I remember right for `options.extractComments` which is a feature of `uglifyjs-webpack-plugin`, we took those defaults from `uglify-es` (default preserved comments). It's better to keep `uglifyjs-webpack-plugin` as near to `uglify-es` defaults as possible even for a small change like this.

**Pull request closed without merging**